### ISSUE TITLE
[Bug]: NativePHP/electron install fails

### ISSUE THREAD

**Issue by phikhi** [NativePHP/laravel#288]
### What were you trying to do?

I was trying to install the nativephp/electron package in a freshly created new Laravel 11 app.

`composer require nativephp/electron`

### What happened?

Installation part went fine.
But when running the composer postAutoloadDump scripts, it throws the following exception :

Log : 

```bash
./composer.json has been updated
Running composer update nativephp/electron
Loading composer repositories with package information
Updating dependencies
Lock file operations: 4 installs, 0 updates, 0 removals
  - Locking nativephp/electron (0.6.2)
  - Locking nativephp/laravel (0.5.1)
  - Locking nativephp/php-bin (0.4.0)
  - Locking spatie/laravel-package-tools (1.16.4)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 4 installs, 0 updates, 0 removals
  - Installing spatie/laravel-package-tools (1.16.4): Extracting archive
  - Installing nativephp/php-bin (0.4.0): Extracting archive
  - Installing nativephp/laravel (0.5.1): Extracting archive
  - Installing nativephp/electron (0.6.2): Extracting archive
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Illuminate\Contracts\Container\BindingResolutionException

  Target [Illuminate\Database\Migrations\MigrationRepositoryInterface] is not instantiable while building [Native\Laravel\Commands\FreshCommand, Illuminate\Database\Migrations\Migrator].

  at vendor/laravel/framework/src/Illuminate/Container/Container.php:1118
    1114▕         } else {
    1115▕             $message = "Target [$concrete] is not instantiable.";
    1116▕         }
    1117▕
  ➜ 1118▕         throw new BindingResolutionException($message);
    1119▕     }
    1120▕
    1121▕     /**
    1122▕      * Throw an exception for an unresolvable primitive.

      +27 vendor frames

  28  artisan:13
      Illuminate\Foundation\Application::handleCommand(Object(Symfony\Component\Console\Input\ArgvInput))

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

### How to reproduce the bug

1. Install a new Laravel 11 app
2. run `composer require nativephp/electron`

### Package Versions

0.6.2

### PHP Version

8.3.6

### Laravel Version

11.6.0

### Node Version

21.7.3

### Which operating systems have you seen this occur on?

macOS

### OS version

macOS 14.4.1 (23E224)

**Comment by simonhamp** [comment]
Fixed in [v0.5.2](https://github.com/NativePHP/laravel/releases/tag/0.5.2)

**Comment by phikhi** [comment]
Thanks a lot !!! 🙏🏻
